src/templates/pycontw-2020/contents/zh/venue.html

```diff
@@ -0,0 +1,109 @@
+{% extends 'contents/_venue.html' %}
+
+{% block title %}會場{% endblock title %}
+
+{% block map_info %}
+
+<h2>
+	<span>國立成功大學</span>
+	<span>國際會議廳</span>
+</h2>
+<p>701 台南市東區大學路 1 號</p>
+
+<script>window.VENUE_NAME = '國立成功大學 國際會議廳'</script>
```

Comment:
It seems to be the Taipei map we used to use. We'll have to change it to the Tainan version. It would be even better if we can mark both NCKU and Good Idea Studio @yychen Do you know where we can make this modification?

Reply:
oh! Actually, I don't know. I need to dig into it. Let me get back to you tomorrow.

Reply:
OK, just edit the javascript file located at `src/static/pycontw-2020/scripts/venue.js`. The coordinates are hard-coded, you can replace the lat/lng with the new value, which should be
```js
const coordinates = [22.997908, 120.215657] // 成大國際會議廳
```
and also replace the corresponding constants inside `centerMap()` function.

I noticed that there's another map that needs to be displayed. So we should make this function more generic. I think you can leave it there, and we'll take over for the adjustment. But you can merely replace the coordinates first if you want to take a look first.

Reply:
@yychen : Thanks for your help! I update the file. Could you please help me review it? Thank you!